timelock-server/src/testCommon/resources/dbTimeLockPaxosMultiServer.ftl

```diff
@@ -0,0 +1,76 @@
+install:
+  paxos:
+    data-directory: "${dataDirectory}"
+    sqlite-persistence:
+      data-directory: "${sqliteDataDirectory}"
+    is-new-service: false
+    leader-mode: ${leaderMode}
+  cluster:
+    cluster:
+      security:
+        trustStorePath: "var/security/trustStore.jks"
+        trustStoreType: "JKS"
+        keyStorePath: "var/security/keyStore.jks"
+        keyStorePassword: "keystore"
+        keyStoreType: "JKS"
+      uris:
+<#list serverProxyPorts as serverProxyPort>
+      - "localhost:${serverProxyPort?c}"
+</#list>
+    local-server: "localhost:${localProxyPort?c}"
+  timestampBoundPersistence:
+    type: database
+    key-value-service:
+      type: "relational"
+      ddl:
+        type: "postgres"
+      connection:
+        type: "postgres"
+        host: "localhost"
+        port: 5432
+        dbName: "atlas"
+        dbLogin: "palantir"
+        dbPassword: "palantir"
+
```

Comment:
Not a big fan of having these constants in 2 places but it may be fine for this test.

Reply:
The docker-compose files are different, so it's in theory possible to change these independently of what you see in the DbKVS tests. It could make sense to extract a common DbKVS TestRule for both the DbKVS and DB-TimeLock tests though I don't think we have the bandwidth for that now.